### PR TITLE
Add withdrawal limits

### PR DESCRIPTION
**Pull request by AntonAndell**

### CODE REVIEW

File: core-contracts/Dex/src/main/java/network/balanced/score/core/dex/DexImpl.java [hunk 1]
```diff
@@ -235,6 +236,7 @@ public void withdraw(Address _token, BigInteger _value) {
         depositDetails.set(sender, deposit_amount.subtract(_value));
 
         Withdraw(_token, sender, _value);
+        BalancedFloorLimits.verifyWithdraw(_token, _value);
```
[the remark this answers is not in the document]
Comment (AntonAndell) [reply core-contracts/Dex/src/main/java/network/balanced/score/core/dex/DexImpl.java:239]: It seems the remove call the withdraw?

[the remark this answers is not in the document]
Comment (ajaya-man) [reply core-contracts/Dex/src/main/java/network/balanced/score/core/dex/DexImpl.java:239]: ahh yeah, the remove calls withdraw eventually which verifies for withdrawn amount.

[the remark this answers is not in the document]
Comment (nightowl121) [reply core-contracts/Dex/src/main/java/network/balanced/score/core/dex/DexImpl.java:239]: yeah the remove calls the withdraw which is floor checked.

**Review by nightowl121 — APPROVED** [review]
Approved

**Pull request merged**